package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "react-typist",
-  "version": "2.0.4",
+  "version": "2.0.5",
```

Comment:
could you remove these changes? I'll update the version when I publish a new version separately